doc/developer/Setup.md

```diff
@@ -81,3 +81,51 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 10. If tests passed, you can now run Mattermost using `make run`
 
 Any issues? Please let us know on our forums at: http://forum.mattermost.org
+
+### Archlinux ###
+
+1. Install Docker
+	1. `pacman -S docker`
+	2. `gpasswd -a user docker`
+	3. `systemctl enable docker.service`
+	4. `systemctl start docker.service`
+	5. `newgrp docker`
+2. Set up your dockerhost address
+	1. Edit your /etc/hosts file to include the following line
+		`127.0.0.1 dockerhost`
+3. Install Go
+	1. `pacman -S go`
+4. Set up your Go workspace and add Go to the PATH
+	1. `mkdir ~/go`
+	2. Add the following to your ~/.bashrc
+		1. `export GOPATH=$HOME/go`
+		2. `export GOROOT=/usr/lib/go`
+		3. `export PATH=$PATH:$GOROOT/bin`
```

Comment:
For me it sets GOROOT automatically when I run `pacman -S go`
Also I think the PATH should be `PATH=$PATH:$GOPATH/bin` as that's the one pacman doesn't setup for you.